integration-tests/mockApis/notificationService.js

```diff
@@ -0,0 +1,20 @@
+const { stubFor } = require('./wiremock')
+
+const stubNotificationService = async () =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: '/notifications-service',
```

Comment:
I can help you with the real URLs

Reply:
This was originally written before the swagger stuff was complete. Updated the service and mocks to reflect the correct ones